wire-ios-sync-engine/WireSyncEngine.xcodeproj/project.pbxproj

```diff
@@ -3504,7 +3377,6 @@
 				EE5AAF3A2874E8C70018FA01 /* BackendEnvironmentProvider+Reachability.swift in Sources */,
 				1634958B1F0254CB004E80DB /* ServerConnection.swift in Sources */,
 				54034F381BB1A6D900F4ED62 /* ZMUserSession+Logs.swift in Sources */,
-				E914CCBC2CDE269B0067F409 /* CreateConversationFolderUseCase.swift in Sources */,
```

Comment:
what happens here? references are deleted ?

Reply:
I converted a sub-tree (Use cases) from groups to folders.
<img width="399" alt="Screenshot 2025-01-21 at 15 25 28" src="https://github.com/user-attachments/assets/d66f2553-b8cc-40fc-9835-40bf680bb4c2" />